plugins/rating/services/RatingService.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Allows user to manipulate their entry rating
+ *
+ * @service rating
+ * @package plugins.rating
+ * @subpackage api.services
+ */
+
+class RatingService extends KalturaBaseService
+{
+	public function initService($serviceId, $serviceName, $actionName)
+	{
+		parent::initService($serviceId, $serviceName, $actionName);
+		
+		if(!RatingPlugin::isAllowedPartner($this->getPartnerId()))
+		{
+			throw new KalturaAPIException(KalturaErrors::FEATURE_FORBIDDEN, RatingPlugin::PLUGIN_NAME);
```

Comment:
Since there not going to be a feature flip on this plugin, we better have an exception of plugin not enabled.

Reply:
@MosheMaorKaltura  there actually will be a feature flip, but it needs to be added on the environment admin.ini file. I'll add to admin.ini.template as well.

Reply:
$ks = $this->getKs();
!$ks || $ks->isAnonymousSession()